doc/source/release/1.15.0-notes.rst

```diff
@@ -291,6 +292,17 @@ and support several Array API compatible array libraries in addition to NumPy
 
 - `scipy.stats.qmc.PoissonDisk` now accepts lower and upper bounds
   parameters ``l_bounds`` and ``u_bounds``.
+- `scipy.stats.fisher_exact` now supports two-dimensional tables with shapes
+  other than ``(2, 2)``.
+- `scipy.stats.truncate` adds the ability to truncate the distribution
+  underlying a random variable and `scipy.stats.abs` provides the ability to
+  take the absolute value of a random variable. The new ``FoldedDistribution``
+  represents the distribution underlying the absolute value of a random
+  variable.
+- Added ``OrderStatisticDistribution``, a class representing the distribution
+  of order statistics of samples drawn from another distribution.
```

Comment:
`OrderStatisticDistribution` isn't public, and the `scipy.stats.order_statistic` factory function is what should be used for this:
https://github.com/scipy/scipy/pull/21790/files

Reply:
```suggestion
```
I'll add this (`order_statistic`) up above.